### PR TITLE
Some minor recommendations

### PR DESCRIPTION
**Pull request by k3yavi**
Hi @jperkel ,

I am making a PR with some minor changes, feel free to selectively merge things.
* I added a gitignore, which helps with vscode to stop tracking metadata files from rust.
* I added explicit versioning for the `gb-io = "0.6.0"`, it's more of a user's choice but I think it's useful to be explicit about the version which was used to compile the program.
* `fn lookup` I made a minor change in the function, as you don't have to change the u8 to char, you can directly compare the u8 based on the ATCG ascii encodings,
* Lastly, I added clap which is very useful command-line argument parser and it display beautiful help messages for the user like following: 

image

Hope these helps ! 

-- Avi

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -35,12 +35,12 @@ const ERR_BAD_NT: usize = 99;
...
-const fn lookup(x: u8) -> usize {
-    match x as char {
-        'T' => 0,
-        'C' => 1,
-        'A' => 2,
-        'G' => 3,
+fn lookup(x: u8) -> usize {
+    match x {
+        65 | 78 => 2,    // A | N 00
+        67 => 1,         // C 01
```
Comment (luizirber) [inline src/main.rs:41]: You can do `b'C'` instead of 67, easier to read =)

**Review by jperkel — APPROVED** [review]
Thanks so much, @k3yavi , these are great! But, why did you modify lookup() to return 2 (A) for N?

File: src/main.rs (hunk 1 above)
Comment (k3yavi) [reply src/main.rs:41]: Yep, I agree, probably it's better to use `b'C'` for better readability. Thanks @luizirber for the comment !

**Comment by k3yavi** [comment]
@jperkel great question about `N`. I should have been more clear about it. Basically `N` is like an ambiguous unknown character. Generally in sequence alignment  related problems `N` is replaced randomly by any of the `ATCG` characters and the aligner would take care of the unknown base through a SNP. It's just better for aligner to handle 4 character through 2 bit-encoding as having 1 more character (N) means having to increase 1 bit per base.  

Here I replaced `N` by `A`, it's obviously not necessary and feel free to change it or remove it all together.

File: src/main.rs (hunk 1 above)
Comment (jperkel) [reply src/main.rs:41]: Done. Thank you @luizirber @k3yavi!

**Pull request merged**